### ISSUE TITLE
Incorrect password exception

### ISSUE THREAD

**Issue by rzylius** [austinsc/node-red-contrib-envisalink-dsc#1]
Hi,

after configuring properties, Incorrect password exception is thrown. I use envisalink4 for DSC alarm, node-red runs on centos7.

regards,

**Comment by austinsc** [comment]
Yeah, I'm getting this now too- I think it has something to do with a firmware update from envisalink.

**Comment by pakerfeldt** [comment]
I'm curious to learn more about this issue. Is it specific for EnvisaLink4? Because I no issues with EnvlisaLink 2DS running with `node-red-contrib-envisalink`.
